inc/extras.php

```diff
@@ -69,3 +69,220 @@ function llorix_one_lite_render_title() {
+				}
+			}
+		}
+	}
+}
+
+
+/**
+ * Register content of sections in polylang.
+ */
+function llorix_one_lite_translations(){
+	$llorix_one_lite_logos = get_theme_mod('llorix_one_lite_logos_content', json_encode( array(
+			array(
+				'image_url' => llorix_one_lite_get_file( '/images/companies/1.png' ) ,
+				'link' => '#',
+				'id' => 'llorix_one_lite_56d069bb8cb71'
+			),
+			array(
+				'image_url' => llorix_one_lite_get_file( '/images/companies/2.png' ) ,
+				'link' => '#',
+				'id' => 'llorix_one_lite_56d069bc8cb72'
+			),
+			array(
+				'image_url' => llorix_one_lite_get_file( '/images/companies/3.png' ) ,
+				'link' => '#',
+				'id' => 'llorix_one_lite_56d069bd8cb73'
+			),
+			array(
+				'image_url' => llorix_one_lite_get_file( '/images/companies/4.png' ) ,
+				'link' => '#',
+				'id' => 'llorix_one_lite_56d06d128cb74'
+			),
+			array(
+				'image_url' => llorix_one_lite_get_file( '/images/companies/5.png' ) ,
+				'link' => '#',
+				'id' => 'llorix_one_lite_56d06d3d8cb75'
+			) ) )
+	);
+	llorix_one_companion_register_string($llorix_one_lite_logos,'logos');
+
+	$llorix_one_lite_contact_info_item = get_theme_mod('llorix_one_lite_contact_info_content', json_encode( array(
+			array(
+				'icon_value' => 'fa-envelope' ,
+				'text' => 'contact@site.com',
+				'link' => '#',
+				'id' => 'llorix_one_lite_56d450a72cb3a'
+			),
+			array(
+				'icon_value' => 'fa-map-marker' ,
+				'text' => 'Company address',
+				'link' => '#',
+				'id' => 'llorix_one_lite_56d069b88cb6f'
+			),
+			array(
+				'icon_value' => 'fa-tablet' ,
+				'text' => '0 332 548 954',
+				'link' => '#',
+				'id' => 'llorix_one_lite_56d069b98cb70'
+			) ) )
+	);
+	llorix_one_companion_register_string($llorix_one_lite_contact_info_item,'contact');
+
+	$llorix_one_lite_footer_social_icons = get_theme_mod('llorix_one_lite_social_icons', json_encode( array(
+		array(
+			'icon_value' => 'fa-facebook' ,
+			'link' => '#',
+			'id' => 'llorix_one_lite_56d069b78cb6e'
+		),
+		array(
+			'icon_value' => 'fa-twitter' ,
+			'link' => '#',
+			'id' => 'llorix_one_lite_56d450842cb39'
+		),
+		array(
+			'icon_value' => 'fa-google-plus-square' ,
+			'link' => '#', 'id' => 'llorix_one_lite_56d450512cb38'
+		) ) )
+	);
+	llorix_one_companion_register_string($llorix_one_lite_footer_social_icons,'footer social icons');
+
+	$llorix_one_lite_header_social_icons = get_theme_mod('llorix_one_lite_very_top_social_icons',json_encode( array(
+			array(
+				'icon_value' => 'fa-facebook' ,
+				'link' => '#',
+				'id' => 'llorix_one_lite_56d069ad8cb6b'
+			),
+			array(
+				'icon_value' => 'fa-twitter' ,
+				'link' => '#',
+				'id' => 'llorix_one_lite_56d069b48cb6c'
+			),
+			array(
+				'icon_value' => 'fa-google-plus-square' ,
+				'link' => '#',
+				'id' => 'llorix_one_lite_56d069b58cb6d'
+			) ) )
+	);
+	llorix_one_companion_register_string($llorix_one_lite_header_social_icons,'header social icons');
+}
+
+include_once( ABSPATH . 'wp-admin/includes/plugin.php' );
```

Comment:
i'm not sure this is the best way to check if the plugin is active. For example, we check if woocommerce is installed like this: https://github.com/Codeinwp/shop-isle-wporg/blob/6b13ab6abd599ec2ee488613059b1df3b82aecc4/inc/functions/extras.php#L45

I think for polylang, you can check if the class Polylang exists